### ISSUE TITLE
TypeError: unsupported operand type(s) for +: 'dict_items' and 'dict_items'

### ISSUE THREAD

**Issue by devopsberlin** [msvbhat/find-unused-reserved-ec2-instances#3]
Please note there is an error in the script, thanks.
```
return dict(unused_reserved_instances_region.items() + unused_reserved_instances.items())
TypeError: unsupported operand type(s) for +: 'dict_items' and 'dict_items'
```

**Comment by gabayliran** [comment]
you can merge two Python dictionaries with a new syntax of Python 3.5:
x = dict(unused_reserved_instances_region.items())
y = dict(unused_reserved_instances.items())
z = {**x, **y}
print("z => ", z)
return z

**Comment by devopsberlin** [comment]
might be a soultion to merge two Python dictionaries in a single expression, a new syntax for this available in Python 3.5, is:

```
    dict1 = unused_reserved_instances_region
    dict2 = unused_reserved_instances

    print("total_unused_reserved_instances dict:")
    total_unused_reserved_instances = {**dict2, **dict1}
    for key, value in total_unused_reserved_instances.items():
        print("Key: ", key)
        print("Value: ", value)

    return(dict(total_unused_reserved_instances))
```
